<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>493</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>333</width>
    <height>493</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>333</width>
    <height>493</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;Configure all settings with what you use in-game&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Fishing key:&lt;/span&gt; the in-game &amp;quot;Use Tool&amp;quot; button. Left-clicking is not supported.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Game resolution:&lt;/span&gt; smaller resolutions have a better performance, and can capture more frames per second. It only works if you are fullscreen or place the game at the top-left corner.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Zoom level:&lt;/span&gt; the 0th zoom level corresponds to the in-game 100% zoom level. And each zoom-in or zoom-out sums or subtracts, respectively, 1 from the zoom level counter. Example: if you play using 80% in-game zoom level, that translates to the -4 zoom level, as you had to click the - button 4 times, starting from the standard zoom level (100%). &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="invalid_settings_label">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="used_key_label_2">
       <property name="text">
        <string>Fishing key:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="change_key">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>1</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Press any key</string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::LogicalMoveStyle</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="res_label_2">
       <property name="text">
        <string>Game resolution:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="res_selection_2">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>Auto</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1280x600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1280x720</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1280x760</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1280x800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1280x960</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1280x1024</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1980x1080</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="zoom_level_label_2">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Zoom level: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="zoom_levelSpinBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>-5</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="save_btn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close_btn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
